### PR TITLE
[Android] MultiplatformBleAdapter integration

### PR DESCRIPTION
**Pull request by bartoszwilk**

### CODE REVIEW

File: android/src/main/java/com/polidea/reactnativeble/RefreshGattMoment.java [hunk 1]
```diff
@@ -1,20 +0,0 @@
-package com.polidea.reactnativeble;
-
-
-enum RefreshGattMoment {
```
Comment (pawelByszewski) [inline android/src/main/java/com/polidea/reactnativeble/RefreshGattMoment.java]: why is it removed?

File: android/src/main/java/com/polidea/reactnativeble/converter/CharacteristicToJsObjectConverter.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        String SERVICE_ID = "serviceID";
+        String SERVICE_UUID = "serviceUUID";
+        String DEVICE_ID = "deviceID";
+        String IS_READABLE = "isReadable";
+        String IS_WRITABLE_WITH_RESPONSE = "isWritableWithResponse";
+        String IS_WRITABLE_WITHOUT_RESPONSE = "isWritableWithoutResponse";
+        String IS_NOTIFIABLE = "isNotifiable";
+        String IS_NOTIFYING = "isNotifying";
+        String IS_INDICATABLE = "isIndicatable";
+        String characteristic = "characteristic";
```
Comment (pawelByszewski) [inline android/src/main/java/com/polidea/reactnativeble/converter/CharacteristicToJsObjectConverter.java]: `String CHARACTERISTIC = "characteristic";` ?

File: android/src/main/java/com/polidea/reactnativeble/RefreshGattMoment.java (hunk 1 above)
Comment (bartoszwilk) [reply android/src/main/java/com/polidea/reactnativeble/RefreshGattMoment.java]: I left this enum inside MBA.

File: android/src/main/java/com/polidea/reactnativeble/converter/CharacteristicToJsObjectConverter.java (hunk 2 above)
Comment (bartoszwilk) [reply android/src/main/java/com/polidea/reactnativeble/converter/CharacteristicToJsObjectConverter.java]: Fixed.

**Pull request merged**